### PR TITLE
Disable Javascript UDFs for 2.51.0

### PR DESCRIPTION
**Pull request by Polber**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by Polber** [comment]
R: @robertwb

File: sdks/python/apache_beam/yaml/yaml_mapping.py [hunk 1]
```diff
@@ -129,14 +149,15 @@ def _as_callable(original_fields, expr, transform_name, language):
 
   _check_mapping_arguments(transform_name, **expr)
 
-  if language == "javascript":
-    return _expand_javascript_mapping_func(original_fields, **expr)
-  elif language == "python":
+  # TODO(yaml) - Disable until JsObjectWrapper can be parsed into Beam Row
+  # if language == "javascript":
```
Comment (robertwb) [inline sdks/python/apache_beam/yaml/yaml_mapping.py:153]: Maybe we could keep this under "javascript-experimental" to keep our tests running? (I would keep the error message below.)

**Review by robertwb — COMMENTED** [review]
Will this not cause several test failures?

File: sdks/python/apache_beam/yaml/yaml_mapping.py (hunk 1 above)
Comment (Polber) [reply sdks/python/apache_beam/yaml/yaml_mapping.py]: Done.

**Comment by damccorm** [comment]
@Polber should we close this since it didn't go in for 2.51?

**Pull request closed without merging**